h2o-py/h2o/sklearn/__init__.py

```diff
@@ -0,0 +1,146 @@
+import inspect
+import sys
+
+from sklearn.base import ClassifierMixin, RegressorMixin
```

Comment:
does this make us dependent on sklearn?

I didn't notice any changes in setup.py

Reply:
@michalkurka only when importing the `h2o.sklearn` module, which is fair enough I guess

Reply:
yes, that makes good sense - if you want to use sklearn functionality - you should have sklearn

makes me think that we should also have a minimal docker image with just the minimal requirements and run smoke tests with just minimum requirements as well